driver-core/src/test/java/com/datastax/driver/core/LargeDataTest.java

```diff
@@ -203,9 +205,10 @@ public void wideRows() throws Throwable {
 
     /**
      * Test a batch that writes a row of size 10,000
+     * Since Cassandra 2.2 large batches are rejected, see CASSANDRA-8011.
```

Comment:
Just a heads up that since CASSANDRA-8011 this test throws InvalidQueryException (batch too large).